lineapy/transformer/node_transformer.py

```diff
@@ -77,7 +77,7 @@ def generic_visit(self, node: ast.AST):
             ast.stmt,
         ):
             if (
-                isinstance(node, ast.FunctionDef)
+                isinstance(node, (ast.FunctionDef, ast.ClassDef, ast.AsyncFunctionDef))
```

Comment:
sorry just for my understanding, this allows us to visit class definitions and treat them as a black box too?

Reply:
We already do all that; this allows us to process decorators on classes as well as async functions.